newsletter-february-2023.rst

```diff
@@ -0,0 +1,119 @@
+- 🚦️ Note from Cloudflare that provides CDN for everything on readthedocs.io:
+
+    On February 14th, 2023,
+    Cloudflare will be doing database maintenance that will impact SSL API availability and may result in certificate issuance delays.
+    The scheduled maintenance will be on February 14, 2023, 14:00 - 16:00 UTC.
+
+.. _rclone: https://rclone.org/
+.. _their announcement: https://www.writethedocs.org/blog/2023-january-update/
+.. _GHSA-5w8m-r7jm-mhp9: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9
+.. _GHSA-hqwg-gjqw-h5wg: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-hqwg-gjqw-h5wg
+.. _GHSA-7fcx-wwr3-99jv: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-7fcx-wwr3-99jv
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+.. raw:: html
+
+   <a href="https://about.readthedocs.com/" style="background-color: #409cff; border-radius: 3px; color: #ffffff; display: block; margin: 30px auto; font-size: 18px; font-weight: 700; line-height: 24px; padding: 15px 0 15px 0; text-align: center; text-decoration: none; width: 238px;">
+     Discover all the features ✨️
+   </a>
+
+Upcoming features
+-----------------
+
+..
+  Notes:
+
+  Next newsletter:
+  Make a general announcement of our Roadmap
+
+  General:
+
+  When creating newsletter drafts, we keep the items here from the previous newsletter.
+  This is in order to ensure due follow-up on features that are announced publicly.
+  
+  Feature done? A great follow-up is to add what was previously an upcoming feature as a released feature in the former section.
+  
+  Feature not done?
+  Make sure that upcoming features are announced with a link to issues or PRs where the progress can be seen.
+  If this is done, then subsequent newsletters aren't compelled to share progress when it's uninteresting.
+  
+  If a feature was announced as upcoming but isn't yet released,
+  then try rephrasing the announcement as a general news update about the progress and where it can be followed.
+
+- More internal components of the build process will have a public API, giving full control and flexibility and allowing for more documentation tools to be used.
+- Our Diátaxis documentation refactor is entering final stages before the first reveal.
+
+Interested in all the details? `View our full Roadmap 📍️`_
+
+.. _View our full Roadmap 📍️: https://github.com/orgs/readthedocs/projects/156/views/4
+
+Possible issues
+---------------
+
+* The change of the standard build output directory will cause issues for anyone that has custom builds and have guessed the old unofficial output directory.
+  See technical details in `readthedocs.org#9888`_
+* If you want to use Sphinx 6 wth sphinx-rtd-theme,
+  we will ship an update next week of both the new theme and an update on our platform dealing with the removal of jQuery from Sphinx.
```

Comment:
```suggestion
  we will ship an update soon of both the new theme and an update on our platform dealing with the removal of jQuery from Sphinx.
```

Reply:
I'll coordinate this with the item below. Releases are shipping today :crossed_fingers:

Reply:
Fixed in a new version